src/Sylius/Behat/Context/Api/SharedStorageChannelContext.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api;
+
+use Sylius\Behat\Service\SharedStorageInterface;
+use Sylius\Component\Channel\Context\ChannelContextInterface;
+use Sylius\Component\Channel\Context\ChannelNotFoundException;
+use Sylius\Component\Channel\Model\ChannelInterface;
+
+class SharedStorageChannelContext implements ChannelContextInterface
+{
+    public function __construct(private SharedStorageInterface $sharedStorage)
+    {
+    }
+
+    public function getChannel(): ChannelInterface
+    {
```

Comment:
The clipboarded channel is utilized in the controller through the test following test: https://github.com/Sylius/Sylius/pull/15598/files#diff-7b8369b44226af38b65d2dc12e402e76da5b0e1de6af1aa90873cf45af8b8149R14
The controller that uses the `channelContext`: https://github.com/Sylius/Sylius/pull/15598/files#diff-9be848d23c251af8c4cbffaea20c334229a34fe8c55fffd9677a615337aea385R38.

Reply:
Ok, didn't see it's an actual `ChannelContext`;
Really shouldn't be in this namespace, maybe move it under `Behat/Service` since that's the usual location for test/helper services